### DISCUSSION TITLE
Background Task System Implementation Step by Step Guide

### DISCUSSION THREAD

**Discussion by ziaatrix** [serenity-is/Serenity#7244] [General]
Hi,

I am new on implementing Background Task System. I could not find in the documentation the full details of Background Task System (https://serenity.is/docs/startsharp/features/background-task-system) 

1. In the project structure, where to create this file?
2. How to set different frequencies like Daily, Hourly and etc.

and the full implementation of this sample.
![image](https://github.com/user-attachments/assets/3b8f41a2-c85c-47ec-8a59-283e5f9075d6)

**Comment by VictorTomaili** [comment]
It is under the StartSharp / Features. So no support from public discussions. You should ask about it in StartSharp repository.
